src/utils/helpers.ts

```diff
@@ -39,6 +39,7 @@ export function sendResponse(
 export function humanizeMilliseconds(milliseconds: number) {
   // Gets ms into seconds
   const time = milliseconds / 1000;
+  if (time < 1) return;
```

Comment:
I would suggest returning "0s" or "1s".
That would make following code redundant
https://github.com/Skillz4Killz/Discordeno-bot-template/blob/987157dd6b5bf4662fc57886892bd38b0e2aae9a/src/inhibitors/cooldown.ts#L23

Reply:
Not a bad idea

Reply:
```suggestion
  if (time < 1) return "1s";
```